lib/src/text_field_props.dart

```diff
@@ -0,0 +1,151 @@
+import 'dart:ui' as ui show BoxHeightStyle, BoxWidthStyle;
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/gestures.dart';
+
+class TextFieldProps {
+  TextFieldProps({
+    TextEditingController? controller,
```

Comment:
here i think you forget `this.controller`

Reply:
Take a look at line [56](https://github.com/salim-lachdhaf/searchable_dropdown/pull/171/files/eb1c435be1e23fe7aca0c83d72b303f1e5b2ead3#diff-cb4a18a31bbe4ea4426a410da67bad2a5563942e042de496fb0c99be182b83a7R56) - here I set a default text controller which is accessible in `_SelectDialogState` which substitutes setting `non empty` controller in previous [implementation](https://github.com/salim-lachdhaf/searchable_dropdown/blob/master/lib/dropdown_search.dart#L509)